simbad/rotsearch/phaser_rotation_search.py

```diff
@@ -0,0 +1,285 @@
+
+    @property
+    def f(self):
+        """The F label from the input hkl"""
+        return self._f
+
+    @f.setter
+    def f(self, f):
+        """Define the F label from the input hkl"""
+        self._f = f
+
+    @property
+    def hires(self):
+        """The high resolution limit of data used to find/refine this solution"""
+        return self._hires
+
+    @hires.setter
+    def hires(self, hires):
+        """Define the high resolution limit of data used to find/refine this solution"""
+        self._hires = hires
+
+    @property
+    def hklin(self):
+        """The input hkl file"""
+        return self._hklin
+
+    @hklin.setter
+    def hklin(self, hklin):
+        """Define the input hkl file"""
+        self._hklin = hklin
+
+    @property
+    def i(self):
+        """The I label from the input hkl"""
+        return self._i
+
+    @i.setter
+    def i(self, i):
+        """Define the I label from the input hkl"""
+        self._i = i
+
+    @property
+    def logfile(self):
+        """The logfile output"""
+        return self._logfile
+
+    @logfile.setter
+    def logfile(self, logfile):
+        """Define the output logfile"""
+        self._logfile = logfile
+
+    @property
+    def nmol(self):
+        """The number of molecules to look for"""
+        return self._nmol
+
+    @nmol.setter
+    def nmol(self, nmol):
+        """Define the number of molecules to look for"""
+        self._nmol = nmol
+
+    @property
+    def pdbin(self):
+        """The input pdb file"""
+        return self._pdbin
+
+    @pdbin.setter
+    def pdbin(self, pdbin):
+        """Define the input pdb file"""
+        self._pdbin = pdbin
+
+    @property
+    def sigf(self):
+        """The SIGF label from the input hkl"""
+        return self._sigf
+
+    @sigf.setter
+    def sigf(self, sigf):
+        """Define the SIGF label from the input hkl"""
+        self._sigf = sigf
+
+    @property
+    def sigi(self):
+        """The SIGI label from the input hkl"""
+        return self._sigi
+
+    @sigi.setter
+    def sigi(self, sigi):
+        """Define the SIGI label from the input hkl"""
+        self._sigi = sigi
+
+    @property
+    def solvent(self):
+        """The estimated solvent content of the crystal"""
+        return self._solvent
+
+    @solvent.setter
+    def solvent(self, solvent):
+        """Define the estimated solvent content of the crystal"""
+        self._solvent = solvent
+
+    @property
+    def timeout(self):
+        """The time in minutes before phaser is killed"""
+        return self._timeout
+
+    @timeout.setter
+    def timeout(self, timeout):
+        """Define the time in minutes before phaser should be killed"""
+        self._timeout = timeout
+
+    def run(self):
+        """Function to run rotation search using PHASER"""
+
+        # Make a note of the current working directory
+        current_work_dir = os.getcwd()
+
+        # Change to the PHASER working directory
```

Comment:
delete

Reply:
All these comments merely state what we can read from the function names anyway ... this adds nothing to the readability of the code and thus adds unnecessary info to the source code. 

Keep your comments basically only to situations where additional explanations are needed, the rest should be clear from good variable, class and function names.